src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -94,91 +102,111 @@
+                                                          (send-utils/calculate-full-route-gas-fee
+                                                           route))
+            fee-in-crypto-formatted                     (when fee-in-native-token
+                                                          (utils/get-standard-crypto-format
+                                                           native-token
+                                                           fee-in-native-token))
+            fee-in-fiat                                 (when-not confirm-disabled?
+                                                          (utils/calculate-token-fiat-value
+                                                           {:currency fiat-currency
+                                                            :balance  fee-in-native-token
+                                                            :token    native-token}))
+            currency-symbol                             (rf/sub [:profile/currency-symbol])
+            fee-formatted                               (when fee-in-fiat
+                                                          (utils/get-standard-fiat-format
+                                                           fee-in-crypto-formatted
+                                                           currency-symbol
+                                                           fee-in-fiat))
+            show-select-asset-sheet                     #(rf/dispatch
+                                                          [:show-bottom-sheet
+                                                           {:content (fn []
+                                                                       [select-asset-bottom-sheet
+                                                                        clear-input!])}])
+            loading-suggested-routes?                   (rf/sub
+                                                         [:wallet/wallet-send-loading-suggested-routes?])
+            sender-network-values                       (rf/sub
+                                                         [:wallet/wallet-send-sender-network-values])
+            suggested-routes                            (rf/sub [:wallet/wallet-send-suggested-routes])
+            routes                                      (when suggested-routes
+                                                          (or (:best suggested-routes) []))
+            no-routes-found?                            (and
+                                                         (every-network-value-is-zero?
+                                                          sender-network-values)
+                                                         (not (nil? routes))
+                                                         (not loading-suggested-routes?))
+            input-error                                 (controlled-input/input-error input-state)]
```

Comment:
@OmarBasem 
Sorry, I don't get it, do you mean that we have events, subs, utils and `def`s defined in the same file?

Reply:
I agree there are lots of var definitions in this file, it is getting bigger and it deserves a refactor.
Not sure if we want to refactor now? Maybe it is too risky as it is a complex and core component for wallet, I would vote for refactor once we have a stable version so we can easily revert commits in case we break things while refactoring. But I am also open to listen to other opinions, no strong opinion on this tbh.